src/components/forms/SignUpForm/SignUpForm.js

```diff
@@ -1,126 +1,153 @@
-/* eslint-disable react/no-unused-state */ // Disabled because state is used and modified in handleChange
-import React, { Component } from 'react';
+import React from 'react';
 import { Button, Col, Form } from 'react-bootstrap';
-import { validateAll } from 'indicative/validator';
+import { connect } from 'react-redux';
+import usersActions from '../../../_actions/users.actions';
 
-class SignUpForm extends Component {
+class SignUpForm extends React.Component {
   constructor(props) {
     super(props);
+    // create a ref to store the confirmed password DOM element
+    this.confirmedPasswordRef = React.createRef();
```

Comment:
Nice!

Reply:
Oh this has now gone because I found I don't need it haha

Reply:
Even better (: